### ISSUE TITLE
Add `--binary-path` when installing

### ISSUE THREAD

**Issue by guestisp** [Neilpang/acme.sh#1149]
Currently, `--upgrade --auto-upgrade 1` install the whole package in acme home dir.

This is good, but sometime, the `acme.sh` script could be placed somewhere else, like in `/usr/local/sbin`

Please add an option to automatically mv/cp the binary to a different location. In example, we always set home dir to /etc/ssl/acme.sh but `acme.sh` script is placed in `/usr/local/sbin` that is inside our `$PATH`

A `--binary-path` would be really usefull.

**Comment by Neilpang** [comment]
can you use  symbol link in /usr/local/sbin/  ?

**Comment by guestisp** [comment]
Yes but I don't like to have server-wide binaries symlinked to a home directory. Usually, is the other way round (an binary from the home symlinked to the server one)

**Comment by RobOfTheSkyPeople** [comment]
Bumping this. This would be helpful. Otherwise, a workaround would be appreciated.

**Comment by RobOfTheSkyPeople** [comment]
Please disregard. I figured out the --home parameter does what we were looking for here.
`./acme.sh --upgrade --auto-upgrade --home /usr/local/share/acme.sh`

I followed https://github.com/lippertmarkus/synology-le-dns-auto-renew which implicitly recommended installing acme.sh in the `/usr/local/share/acme.sh` directory. 

I'll be re-evaluating the wisdom of installing acme.sh and storing certs in general and unencrypted keys in particular in a shared location with these permissions:
`drwxrwxrwx certadmin users`

versus the home directory with these permissions
`drwx--x--x+ certadmin users`

Follow that discussion here: https://github.com/lippertmarkus/synology-le-dns-auto-renew/issues/2

**Comment by OverengineeredNetwork** [comment]
Hi @guestisp 

I'm revisiting this for myself. Were you able to try the `--home` parameter to specify a different upgrade path? 

I have uninstalled and re-installed acme.sh multiple times, so I'm not sure if this works.

**Comment by neilpang** [comment]
acme.sh is not a single relocatable binary -- it needs its home directory (dnsapi/, deploy/, conf). Copying the script elsewhere breaks `--upgrade`. The supported way to get a stable path in $PATH is a symlink: `ln -s /etc/ssl/acme.sh/acme.sh /usr/local/sbin/acme.sh` -- it survives auto-upgrades because the target file is updated in place. Closing.
